deployment/CP01CNSP_Deploy.csv

```diff
@@ -1,10 +1,37 @@
+AT27,,AT27,,CP01CNSP-SP001-08-CTDPFJ000,2,1,2015-05-08T01:28:30,2015-05-10T16:11:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-CTDPFJ-00347,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-09-FLORTJ000,2,1,2015-05-08T01:28:30,2015-05-10T16:11:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-FLORTJ-01205,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-10-PARADJ000,2,1,2015-05-08T01:28:30,2015-05-10T16:11:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-PARADJ-00437,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-00-SPPENG000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGCON-CSPPCP-00005,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-02-OPTAAJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-OPTAAJ-00181,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-03-NUTNRJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-NUTNRJ-00428,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-05-VELPTJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-VELPTJ-11915,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-06-DOSTAJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-DOSTAJ-00351,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-07-SPKIRJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-SPKIRJ-00265,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-08-CTDPFJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-CTDPFJ-00347,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-09-FLORTJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-FLORTJ-01205,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-10-PARADJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-PARADJ-00437,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+SI20150722,,MGL1512,,CP01CNSP-SP001-00-SPPENG000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGCON-CSPPCP-00005,40.13408,-70.77013,,101,134,Instrument detached and on surface; recovered by SOOP

```

Comment:
Should this say "Profiling body detached..."  the instruments didn't detach, right?

Reply:
Yeah "profiling body detached" is better. I mis-read a line in the FAR (3111-00038) that made me think we called the profiling body the "instrument".